include/aspect/material_model/interface.h

```diff
@@ -217,6 +217,16 @@ namespace aspect
                           const bool use_strain_rates = true);
 
       /**
+       * Copy Constructor.
+       */
+      MaterialModelInputs(const MaterialModelInputs<dim> &material);
+
+      /**
+       * Assignment operator.
+       */
+      MaterialModelInputs &operator=(const MaterialModelInputs &material);
```

Comment:
Hm, but then I'm curious again how this worked before. If you remove the operator, what are the errors you get from the existing code?

I'm not saying that you should revert the patch, I'm just curious to see what changed that required you to add the copy operator. I must be missing something that it's not clear to me why you need it.